org/views.py

```diff
@@ -93,3 +93,83 @@ def AddVolunteer(request,org_id):
             return Response({"message":"You are added as a volunteer"},status.HTTP_201_CREATED)
     else:
         return Response({"detail":"Organization not present"},status.HTTP_400_BAD_REQUEST)
+
+
+@swagger_auto_schema(
+    operation_id="view_list_of_permissions",
+    method='GET',
+    responses={
+        '200': set_example(responses.list_permission_set_200),
+        '401': set_example(responses.user_unauthorized_401),
+        '404': set_example({"detail": "This organisation doesn't exist."}),
+        '400': set_example({"detail" : "You are not a member of this organisation"}),
+        '403': set_example({"detail": "You are not authorised to view this."}),
+    }
+)
+@api_view(['GET'])
+@permission_classes([IsAuthenticated])
+def Permission_Set_List(request,org_id):
```

Comment:
`Permission_Set_List` view has been deleted from the `dev` branch and should not be added by this PR. Please delete this view.

Reply:
ok